src/Signup.js

```diff
@@ -11,7 +11,7 @@ function Signup() {
       <Form onSubmit={signUpUserFirebase}>
         <Form.Group controlId="formUsername">
           <Form.Label>Username</Form.Label>
-          <Form.Control type="email" name="username" placeholder="Enter username" />
+          <Form.Control type="username" name="username" placeholder="Enter username" />
```

Comment:
Good fix! I think we'll be shifting to email only accounts but we'll discuss this as a team.